.zenodo.json

```diff
@@ -51,6 +56,26 @@
             "orcid": "0000-0002-0922-5770"
         }
     ],
+    "contributors": [
+        {
+            "affiliation": "Scientific Software Engineering Center at JHU",
+            "name": "Jane Doe",
+            "orcid": "0000-0000-0000-0000",
+            "type": "ProjectManager"
+        },
+        {
+            "affiliation": "Scientific Software Engineering Center at JHU",
+            "name": "John Doe",
+            "orcid": "0000-0000-0000-0000",
+            "type": "Sponsor"
+        }
+    ],
+    "grants": [
+        {"": ""}
+    ],
+    "references": [
+        "https://github.com/ssec-jhu/base-template 10.5281/zenodo.14052740."
+    ],
     "description": "Accelerating Software Development for Science Research",
     "license": "other-open",
```

Comment:
@jamienoss 

I think we can be more specific if we want with the license to match the license file. To match the LICENSE file we can change `"other-open"` to `"bsd-3-clause"`, which seems to be the id associated with the license:

https://zenodo.org/api/vocabularies/licenses?q=bsd-3-clause

No worries if you want to leave it as is.

Reply:
Adding that now, thanks for finding that 👍

Reply:
@ryanhausen back to you.